packages/lab/src/pill-next/PillNext.tsx

```diff
@@ -29,39 +25,27 @@ export const PillNext = forwardRef<HTMLButtonElement, PillNextProps>(
     });
     const { buttonProps, active } = useButton<HTMLButtonElement>({
       disabled,
+      onClick,
       ...restProps,
```

Comment:
This can create confusion, we should replace restProps with the specific props useButton accepts